hdr/libccc/format.h

```diff
@@ -43,6 +43,7 @@
 #endif
 //!@}
 
+#include <inttypes.h>
```

Comment:
I will probably need to add some code to define the `PRIxx` in the case when the `__NOSTD__` macro is active. (Notice how any standard header included will always be in a `#ifndef __NOSTD__` block)

Reply:
right, that is a good point

Reply:
ok, done via 9db813490cf69cc29db86711776b5aa32c25c20c